jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -31,7 +31,9 @@ YUI.add('entity-details', function() {
     fetchSuccess: function(models) {
       this.setState({waitingForFetch: false});
       if (models.length > 0) {
-        this.setState({entity: models[0].toEntity()});
+        var entity = models[0];
+        this.setState({entityModel: entity});
+        this.setState({entity: entity.toEntity()});
```

Comment:
I'd rather just store the model in state and then convert it to a POJO via the toEntity function in the `render` function. One source of truth.